### ISSUE TITLE
CMake error when following tutorial on Windows 10: "Could not find a package configuration file <package>"

### ISSUE THREAD

**Issue by dgellow** [microsoft/vcpkg#13742]
**Describe the bug**

I'm on Windows 10, trying to learn to use vcpkg by following the [SQLite example](https://github.com/microsoft/vcpkg/blob/master/docs/examples/installing-and-using-packages.md). I cannot make it work with CMake, installed dependencies cannot be found.

**Environment**
- OS: Windows 10, Dev channel (OS Build 20221.1000)
- Compiler:
```
PS > cl.exe
Microsoft (R) C/C++ Optimizing Compiler Version 19.27.29111 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
```

**To Reproduce**

1. `git clone https://github.com/microsoft/vcpkg`
2. `cd vcpkg`
3. `.\bootstrap-vcpkg.bat`
4. `.\vcpkg.exe install sqlite3`

```
PS > .\vcpkg.exe install sqlite3
# removed the extra output logs for readability
Building package sqlite3[core]:x86-windows... done
Installing package sqlite3[core]:x86-windows...
Installing package sqlite3[core]:x86-windows... done
Elapsed time for package sqlite3:x86-windows: 51.39 s

Total elapsed time: 1.731 min

The package sqlite3:x86-windows provides CMake targets:

    find_package(sqlite3 CONFIG REQUIRED)
    target_link_libraries(main PRIVATE sqlite3)
```

5. Now I create a new project directory, add a `CMakeList.txt` and `main.cpp`

```cmake
# CMakeLists.txt
cmake_minimum_required(VERSION 3.0)
project(test)

find_package(unofficial-sqlite3 CONFIG REQUIRED)

add_executable(main main.cpp)

target_link_libraries(main PRIVATE unofficial::sqlite3::sqlite3)
```

```cpp
// main.cpp
#include <sqlite3.h>
#include <stdio.h>

int main()
{
    printf("%s\n", sqlite3_libversion());
    return 0;
}
```

6. I try to build

```
PS >  cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=C:/Users/Sam/Development/vcpkg/scripts/buildsystems/vcpkg.cmake
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.20221.
CMake Error at C:/Users/Sam/Development/vcpkg/scripts/buildsystems/vcpkg.cmake:441 (_find_package):
  Could not find a package configuration file provided by
  "unofficial-sqlite3" with any of the following names:

    unofficial-sqlite3Config.cmake
    unofficial-sqlite3-config.cmake

  Add the installation prefix of "unofficial-sqlite3" to CMAKE_PREFIX_PATH or
  set "unofficial-sqlite3_DIR" to a directory containing one of the above
  files.  If "unofficial-sqlite3" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:4 (find_package)


-- Configuring incomplete, errors occurred!
See also "C:/Users/Sam/Development/temp/CMakeHelloWorld/build/CMakeFiles/CMakeOutput.log".
```

**Expected behavior**

I expect installed dependencies to be found and the build to succeed.

**Failure logs** 

- [CMakeOutput.log](https://github.com/microsoft/vcpkg/files/5282960/CMakeOutput.log)

**Comment by Hoikas** [comment]
It looks like the example documentation is out of date. According to the console output:
 ```plain
The package sqlite3:x86-windows provides CMake targets:

    find_package(sqlite3 CONFIG REQUIRED)
    target_link_libraries(main PRIVATE sqlite3)
```

So, your CMake list should be:
```cmake
cmake_minimum_required(VERSION 3.0)
project(test)

find_package(sqlite3 CONFIG REQUIRED)

add_executable(main main.cpp)

target_link_libraries(main PRIVATE sqlite3)
```

**Comment by dgellow** [comment]
Ok, I found something that works for me:

1. `.\vcpkg.exe install sqlite:x64-windows`. I understand now that by default the selected triplet is `x86-windows`, that's a bit counter intuitive IMHO, I guess I went too fast when following the guide as I see that it is clearly specified their.
2. For some reasons the guide uses `unofficial-sqlite3` and `unofficial::sqlite3::sqlite3`. I don't know why they use this weird namespace, but in my case I had to use `sqlite3` directly for both `find_package()` and `target_link_libraries()`.
